### PR TITLE
Update progeny dep

### PR DESCRIPTION
**Pull request by m4dz**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,7 +10,9 @@ function StylusCompiler(cfg) {
   this.rootPath = cfg.paths.root;
   this.config = (cfg.plugins && cfg.plugins.stylus) || {};
   this.getDependencies = progeny({
-    rootPath: this.rootPath
+    rootPath: this.rootPath,
+    extension: this.extension,
```
Comment (es128) [inline index.js]: is specifying `extension` necessary? won't progeny auto-detect it from the filepath?

I suppose it won't do any harm though

**Comment by es128** [comment]
Also, this PR may want to wait for the updates to progeny, which will be at least a minor version bump (semver caret on 0. versions won't bump minor automatically)

**Comment by m4dz** [comment]
> this PR may want to wait for the updates to progeny

Of course :smile:

**Comment by kaizau** [comment]
Here's another blocker — hopefully a quick fix: https://github.com/es128/progeny/issues/14

With `reverseArgs: true` and `"progeny": "es128/progeny"`, stylus-brunch is recompiling on import changes as expected.

**Pull request closed without merging**